pytests/test_pyhelios.py

```diff
@@ -100,3 +246,36 @@ def test_detector(test_sim):
     scene_file = find_scene(path_to_survey)
     if os.path.isfile(scene_file):
         os.remove(scene_file)
+
+
+def test_output():
+    """Validating the output of a survey started with pyhelios"""
+    from pyhelios import SimulationBuilder
+    survey_path = Path('data') / 'test' / 'als_hd_demo_tiff_min.xml'
+    pyhelios.setDefaultRandomnessGeneratorSeed("43")
+    simB = SimulationBuilder(
+        surveyPath=str(survey_path.absolute()),
+        assetsDir=WORKING_DIR + os.sep + 'assets' + os.sep,
+        outputDir=WORKING_DIR + os.sep + 'output' + os.sep,
+    )
+    simB.setFinalOutput(True)
+    simB.setRebuildScene(True)
+    simB.setCallbackFrequency(100)
+    simB.setNumThreads(1)
+    simB.setKDTJobs(1)
+
+    sim = simB.build()
+
+    sim.start()
+    output = sim.join()
+    measurements_array, trajectory_array = pyhelios.outputToNumpy(output)
+    # works locally
+    # np.testing.assert_allclose(measurements_array[0, :3], np.array([474500.3, 5473529.0, 106.0196]), rtol=0.000001)
+    # works in GitHub actions
+    np.testing.assert_allclose(measurements_array[0, :3], np.array([474500.3, 5473530.0, 106.0587]), rtol=0.000001)
```

Comment:
Can you explain why there are 4cm difference between the Windows and the Linux implementation? This is a large enough difference to make me worry.

Reply:
i.e., if the values in the comment are correct, we have to investigate this difference @albertoesmp

Reply:
> I have the same issue with `PermissionError` in some of my tests. @albertoesmp could this be related to the problem of renaming `.las` Files after creation (#209), and perhaps only a Windows problem?

Maybe the files are not properly closed in Windows. Or maybe the permissions must be explicitly set for any file created by HELIOS++. The first makes more sense to me because now we have "yielders" to prevent multiple small writings.

Can you run an `attrib` command from the Windows console on both a file that you can delete and one that you cannot? I would like to see if there is any difference.

Does it happen with the files related to the last leg only?

Reply:
Yes, in the two cases in this test file, it even happened for the trajectory .txt file only